dist/index.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
```

Comment:
Please make sure there is no linter error in the PR.

Reply:
Thank you @SajjadAhmad14. I have been debugging the Lighthouse linter, and still, it fails to show me the actual errors. I'll provide an image of a report on my machine, generated by Lighthouse. 
![image](https://user-images.githubusercontent.com/35552263/165639470-18d070ae-0563-4a77-a0f9-f3107c8b89bd.png)
What can I do about this?